developer/src/inst/build.sh

```diff
@@ -105,6 +111,20 @@ function verify-installer-signatures() {
   verify-all-executable-signatures-in-folder "$DEVELOPER_ROOT/release/${VERSION}"
 }
 
+function verify-node-installer-version() {
+  builder_heading verify-node-installer-version
+
+  local REQUIRED_NODE_VERSION=$("$JQ" -r .engines.node < "$KEYMAN_ROOT/package.json")
+  local INSTALLER_NODE_VERSION="$("$THIS_SCRIPT_PATH/node/dist/node.exe" --version)"
+  INSTALLER_NODE_VERSION="${INSTALLER_NODE_VERSION##v}"
+
+  if [[ "$REQUIRED_NODE_VERSION" != "$INSTALLER_NODE_VERSION" ]]; then
+    builder_echo error "Installer node version in /developer/src/inst/node/dist/node.exe is '$INSTALLER_NODE_VERSION',"
+    builder_echo error "but the expected version per /package.json is '$REQUIRED_NODE_VERSION'."
+    builder_die "Version mismatch for bundled node.exe version"
```

Comment:
What if verify-node-installer-version() returned a status ?
0 - versions match
1 - versions don't match

And have the caller decide when to builder_die.

do_test() could strictly test and echo the result, while do_publish would fail the build.

Reply:
I don't see any real benefit to this -- AFAICT `do_test()` should die if tests fail, just like all other builder tests, so might as well leave the code in `verify-node-installer-version()`